<launch>
  <node pkg="wall_following" type="detect" name="map_based_detect" output="screen">
  	<param name="least_n"        type="int"    value="20"         />
  	<param name="angle_limit"    type="double" value="0.01"       />
  	<param name="laser_frame_id" type="str"    value="base_scan"  />
  	<param name="base_frame_id"  type="str"    value="base_link"  />
  	<param name="map_frame_id"   type="str"    value="map"        />
  	<param name="update_map_dif" type="int"    value="10"         />
  	<param name="skip_interval"  type="double" value="0.02"       />
  </node>



</launch>